utils/swift_build_support/swift_build_support/productpipeline_list_builder.py

```diff
@@ -137,11 +141,11 @@ def infer(self):
                             p.get_dependencies()))
 
         for i in range(len(inferred_pipeline_list)):
-            pipeline = inferred_pipeline_list[i]
+            inferred_pipeline = inferred_pipeline_list[i]
```

Comment:
renaming `pipeline` here as otherwise we essentially rebound this variable with values of different type, which I don't think plays well with type annotations